flake.nix

```diff
@@ -19,6 +19,10 @@
       url = "github:Kleidukos/get-tested/v0.1.6.0";
       flake = false;
     };
+    weeder-src = {
+      url = "github:ocharles/weeder";
+      flake = false;
```

Comment:
In fact, it is actually a [flake](https://github.com/ocharles/weeder/blob/master/flake.nix) :pled:
Sadly, it doesn't expose executable